src/nameable.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './add'
+
+class Nameable < Person
+  def correct_name
```

Comment:
- [ ] Nameable is not a child class of Person class. Kindly remove that `< Person` to make your application work as expected :blush:

Reply:
Thank you so much, actually, I had a couple of problems because of that, thank you for giving me the hint, I was able to resolve it @HAMMAS-SALEEM .